src/base-apis.js

```diff
@@ -678,6 +678,27 @@ MatrixBaseApis.prototype.setRoomDirectoryVisibilityAppService =
     );
 };
 
+// User Directory Operations
+// =========================
+
+/**
+ * Query the user directory with a term matching user IDs, display names and domains.
+ * @param {object} opts options
+ * @param {string} opts.term the term with which to search.
+ * @param {number=} opts.limit the maximum number of results to return. Defaults to 10.
```

Comment:
think it would be clearer to say something like "the server will apply a limit if unspecified". I don't think you need to make it explicit what the limit is.